tools/spec-gen-sdk/src/cli/cli.ts

```diff
@@ -186,6 +189,11 @@ yargs(hideBin(process.argv))
           description: "The run mode of the tool, allowed values are 'local','spec-pull-request','release','batch'",
           default: "local",
           choices: ['local', 'spec-pull-request', 'release', 'batch'],
+        },
+        'skip-sdk-gen-from-openapi': {
```

Comment:
Couldn't we accomplish this also by not passing in the readme? Or do you still want to validate the readme for some reason?

Reply:
Validating SDK configuration is in the `spec-gen-sdk` tool itself. Runner has no idea about it and just blindly passes them in.